docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -89,54 +139,68 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
     python epicurious_recipes_import.py
 
-Sample queries with HTTP client
--------------------------------
 
-With the data in place, we can start trying some queries against your OpenSearch service. Since it has a simple HTTP interface, you can use your favorite HTTP client. In these examples, we will use `httpie <https://github.com/httpie/httpie>`_ because it's one of our favorites.
+.. _get-mapping-with-python:
 
-First, export the ``SERVICE_URI`` variable with your OpenSearch service URI address and index name from the previous script:
+Get data mapping with Python
```

Comment:
With data mapping examples for both JS and Python the article grew. Let's move "Get data mapping" (with examples of JS and Python) to a separate article? and add a reference in this article.

Reply:
I agree but imo a new structure maybe fit more in a different PR because the issue is happening in other places, so it would be nice if we can discuss in this issue first:
https://github.com/aiven/devportal/issues/591

Just some rough ideas to what we think is the best for structuring this part.